TelevisionAcademyIOSPlayerPlugin.podspec

```diff
@@ -15,10 +15,10 @@ Pod::Spec.new do |s|
     s.requires_arc = true
     s.swift_version = '5.0'
     s.static_framework = true
-    s.resources = ['ios/TelevisionAcademyIOSPlayerPlugin/Resources/Images/*.png', 'ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/*.{xib,nib,storyboard}']
-    s.source_files = 'ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/**/*.{swift,h,m}'
```

Comment:
You will need to look recursvely in the folders, as not all your classes are under PluginClasses (they are in subfolders).

Reply:
Added subfolders /**/*.swift